### ISSUE TITLE
Any way to extend the container and use Node.js 8.10.0?

### ISSUE THREAD

**Issue by lmammino** [iopipe/awslambda-npm-install#2]
Currently trying to do this:

```Dockerfile
ARG NODE_VERSION=8.10.0
FROM iopipe/awslambda-npm-install

# do extra stuff to build my container
```

This doesn't seem to be working and I still get Node.js 6.10 in the final container.

Is there any way to override the `NODE_VERSION` arg when extending from the iopipe container?

**Comment by kolanos** [comment]
The `FROM` line should be first.

**Comment by ewindisch** [comment]
The ARG is a buildarg, so you could build this project as such:

`docker build -t  lmammino/awslambda-npm-install:8.10.0 --build-arg NODE_VERSION=8.10.0 git://github.com/iopipe/awslambda-npm-install`

then use:

```
FROM lmammino/awslambda-npm-install:8.10.0
```

I'll look at creating an official build for 8.10.0 for `iopipe/aws-npm-install` however!

**Comment by ewindisch** [comment]
@lmammino I've published an image for 8.10.0 (it is now the default)

Documentation is also updated. #3

**Comment by lmammino** [comment]
Awesome, thanks a million @ewindisch :)
